src/v2/Apps/Feature/Components/FeatureHeader.tsx

```diff
@@ -11,7 +11,7 @@ import {
 } from "@artsy/palette"
 import { createFragmentContainer, graphql } from "react-relay"
 import { FeatureHeader_feature } from "v2/__generated__/FeatureHeader_feature.graphql"
-import { NavBarHeight } from "v2/Components/NavBar"
+import { NAV_BAR_HEIGHT } from "v2/Components/NavBar"
```

Comment:
Renamed to just be constant case because this looks like a component at a glance.

Reply:
I call these Alloy consts 😄